models/commons/src/reference/presets/_shared/reference-presets-builder.spec.ts

```diff
@@ -0,0 +1,50 @@
+import { buildReferencePresets } from './reference-presets-builder';
+
+const presets = buildReferencePresets('associate-role');
+
+describe('AssociateRoleReference preset', () => {
```

Comment:
I assume this test is a stand in for the tests for each reference preset type?  Which is fine, and I'm fine with only testing one preset as a stand in for all of them.

but if the test ends up needing to be more comprehensive for whatever reason it would be pretty straightforward to loop over every reference type in `models/commons/src/reference/constants.ts` using jests [test.each](https://jestjs.io/docs/api#testeachtablename-fn-timeout)

Reply:
Thanks for the suggestion.

My reasoning for having just one test file is that we're not testing the presets but the underlying helper (`reference-presets-builder`) those presets use.
Testing each presets does not seem very valuable to me as the only think it changes is the name of the `typeId` and we're already using a constant for populating that property.